sw/ground_segment/cockpit/live.ml

```diff
@@ -1521,6 +1532,45 @@ let get_shapes = fun (geomap:G.widget)_sender vs ->
 let listen_shapes = fun (geomap:G.widget) ->
   safe_bind "SHAPE" (get_shapes geomap)
 
+let get_cam_status = fun (geomap:MapCanvas.widget) _sender vs ->
```

Comment:
If a point can't be plotted, the GCS will stop updating the footprint and retain the last plotted footprint. Once the aircraft has leveled off and a footprint can be plotted, then the footprint continues to update. I have performed multiple flights with this code without any GCS error messages or crashing.